discretisedfield/mesh.py

```diff
@@ -950,6 +950,84 @@ def plane(self, *args, n=None, **kwargs):
 
         return plane_mesh
 
+    def sel(self, *args, **kwargs):
+        no_book_keeping = False
+        if len(args) > 1 or len(kwargs) > 1:
+            raise ValueError("Select method only accepts one dimension at a time.")
+
+        if args and not kwargs:
+            dim = args[0]
+            range_ = None
+        elif not args and kwargs:
+            dim, range_ = list(kwargs.items())[0]
+        else:
+            raise ValueError("No argument passed.")
```

Comment:
This would also happen when we pass args and kwargs (one each). So maybe a different error message would be better?